src/main/kotlin/blackjack/domain/BlackjackGame.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.domain
+
+import blackjack.domain.card.CardDeck
+import blackjack.domain.player.Player
+import blackjack.domain.player.Players
+
+class BlackjackGame(val players: Players, val deck: CardDeck) {
```

Comment:
```suggestion
class BlackjackGame(val players: Players, val deck: CardDeck = CardDeck()) {
```

default value를 사용하면 어떨까요?